src/main/resources/assets/computercraft/lua/rom/apis/vector

```diff
@@ -76,9 +76,9 @@ local vmetatable = {
 
 function new( x, y, z )
 	local v = {
-		x = x or 0,
-		y = y or 0,
-		z = z or 0
+		x = type(x) == "number" and x or 0,
```

Comment:
Make this change and i'll merge the patch. I'm generally ok with letting people misuse things if it only makes their *own* computer crash, but this is elegant.